distro_test.go

```diff
@@ -112,10 +112,13 @@ func TestDefaultDistro(t *testing.T) {
 
 	testCases := map[string]struct {
 		registryInaccessible bool
+		dontCreateDistro     bool
 
+		wantOK  bool
 		wantErr bool
 	}{
-		"Success": {},
+		"Success":                 {wantOK: true},
+		"Success with no distros": {dontCreateDistro: true},
```

Comment:
it’s "Success even if the distro does not exist", no?

Reply:
Sorry, I meant a test case where we set a default distro to a distro that doesn’t exist?